cmd/cluster-operator/README.md

```diff
@@ -16,7 +16,7 @@ kind load docker-image ghcr.io/kurator-dev/cluster-operator:0.3-dev --name <your
 helm repo add jetstack https://charts.jetstack.io
 helm repo update
 kubectl create namespace cert-manager
-helm install -n cert-manager cert-manager jetstack/cert-manager --set installCRDs=true
```

Comment:
Can you add "--version" to use the specified version

Reply:
version v1.15.3 has been specified